src/cloud-cli/applications/configure.ts

```diff
@@ -0,0 +1,57 @@
+import axios from "axios";
+import fs from "fs";
+import YAML from "yaml";
+import { createGlobalLogger } from "../../telemetry/logs";
+import { getCloudCredentials } from "../utils";
+import { ConfigFile, loadConfigFile, operonConfigFilePath } from "../../operon-runtime/config";
+
+export async function configureApp(host: string, port: string, dbName: string) {
+    const logger = createGlobalLogger();
+    const userCredentials = getCloudCredentials();
+    const bearerToken = "Bearer " + userCredentials.token;
+    
+    // call cloud and get hostname and port
+    const res = await axios.get(`http://${host}:${port}/${userCredentials.userName}/databases/userdb/info/${dbName}`, 
+    {
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: bearerToken,
+      },
+    });
+
+    // if status is not available or no hostname/port print error and exit
+    if (res.status != axios.HttpStatusCode.Ok) {
```

Comment:
This PR seems to have some linter errors.

Reply:
fixed